test/Makefile

```
@@ -18,10 +18,13 @@ all: test
 
 NON_INTERACTIVE_FLAG=--skip-resource=false --skip-controller=false --skip-admission-controller=false
 
-test: build
+test: build check
 	go test ./pkg/...
 	bash -c "find pkg/apis/ -name apiserver.local.config | xargs rm -rf"
 
+check: build
+	go vet $$(go list ./... | grep -vE "_generated" | grep -vE "plugin/admission")
```

sanitizing all _generated files is probably overkilling, i suggest:

```suggestion
	go vet $$(go list ./... | grep -vE '(clientset|listers|informers)_generated'
```